tests/topotests/bgp_prefix_sid2/r1/bgpd.conf

```diff
@@ -21,5 +21,7 @@ router bgp 1
  !
  address-family ipv6 vpn
   neighbor 10.0.0.101 activate
+  neighbor 10.0.0.101 route-map DENY_ALL out
```

Comment:
Is this on purpose?

Reply:
Yes it is. See commit log https://github.com/FRRouting/frr/pull/14890/commits/ebc71fed6a9f46e1b7bc50e399b7ee87c41cb4da